bigbluebutton-html5/imports/ui/components/whiteboard/slide-controls/component.jsx

```diff
@@ -0,0 +1,296 @@
+          disabled={this.prevSlideAvailable() ? false : true}
+          color={'default'}
+          icon={'left-arrow'}
+          size={'md'}
+          onClick={this.previousSlide}
+          label={'Previous Slide'}
+          hideLabel={true}
+          className={styles.prevSlide}
+        />
+        {/*Next Slide button*/}
+        <Button
+          role="button"
+          aria-labelledby="nextSlideLabel"
+          aria-describedby="nextSlideDescrip"
+          aria-controls="skipSlide slideComponent"
+          disabled={this.nextSlideAvailable() ? false : true}
+          color={'default'}
+          icon={'right-arrow'}
+          size={'md'}
+          onClick={this.nextSlide}
+          label={'Next Slide'}
+          hideLabel={true}
+        />
+        {/*Skip Slide drop down*/}
+        <select
+          id="skipSlide"
+          role="listbox"
+          aria-labelledby="skipSlideLabel"
+          aria-describedby="skipSlideDescrip"
+          aria-controls="slideComponent"
+          aria-live="polite"
+          aria-relevant="all"
+          value={currentSlideNum}
+          onChange={this.skipToSlide}
+          className={styles.skipSlide}
+        >
+          {this.renderSkipSlideOpts(numberOfSlides)}
+        </select>
+        {/*Fit to width button*/}
+        <Button
+          role="button"
+          aria-labelledby="fitWidthLabel"
+          aria-describedby="fitWidthDescrip"
+          color={'default'}
+          icon={'fit-to-width'}
+          size={'md'}
+          circle={false}
+          onClick={this.clickHandler}
+          label={'Fit to Width'}
+          hideLabel={true}
+        />
+        {/*Fit to screen button*/}
+        <Button
+          role="button"
+          aria-labelledby="fitScreenLabel"
+          aria-describedby="fitScreenDescrip"
+          color={'default'}
+          icon={'fit-to-screen'}
+          size={'md'}
+          circle={false}
+          onClick={this.clickHandler}
+          label={'Fit to Screen'}
+          hideLabel={true}
+        />
+        {/*Zoom slider*/}
+        <form
+          className={classNames(styles.zoomForm, { [styles.zoomFormNoBorder]: true })}
+        >
+          <div className={styles.zoomMinMax}> 100% </div>
+          <input
+            role="slider"
+            aria-labelledby="zoomLabel"
+            aria-describedby="zoomDescrip"
+            aria-valuemax="400"
+            aria-valuemin="100"
+            aria-valuenow={this.state.sliderValue}
+            value={this.state.sliderValue}
+            step="5"
+            type="range"
+            min="100"
+            max="400"
+            onChange={this.handleValuesChange}
+            onInput={this.handleValuesChange}
+            className={styles.zoomSlider}
+          />
+          <div className={styles.zoomMinMax}> 400% </div>
+        </form>
+      </div>
+     );
+  }
+
+  renderAriaLabelsDescs() {
+    return (
+      <div className={styles.ariaLabelsDescs}>
```

Comment:
For later reference, the solution was: HTML attribute `hidden` was added instead, and className was removed (`hidden` applies css itself)